src/yukon/backend/src/devserv/description/template/sess_description.json

```diff
@@ -0,0 +1,61 @@
+{
+        "log_file": "",
+        "interactions": [{
+                "bus": {
+                        "id": ""
+                },
+                "nodes": {
+                        "0": {
+                                "subject": {
+                                        "publish": {
+                                                "timestamp_start": "",
+                                                "timestamp_end": "",
+                                                "port_id": "",
+                                                "timeout": ""
+                                        },
+                                        "subscribe": {
+                                                "timestamp_start": "",
+                                                "timestamp_end": "",
+                                                "port_id": "",
+                                                "timeout": ""
+                                        }
+                                },
+                                "service": {
+                                        "send": {
+                                                "timestamp": "",
+                                                "port_id": ""
+                                        }
+                                }
+
+                        },
+                        "1": {
+                                "subject": {
+                                        "publish": {
+                                                "timestamp_start": "",
+                                                "timestamp_end": "",
+                                                "port_id": "",
+                                                "timeout": ""
+                                        },
+                                        "subscribe": {
+                                                "timestamp_start": "",
+                                                "timestamp_end": "",
+                                                "port_id": "",
+                                                "timeout": ""
+                                        }
+                                }
+
+                        }
+                }
+        }],
+        "failure_injection": [{
+                        "number": "",
+                        "bus": "",
```

Comment:
Maybe we could use a less ambiguous term here, like `redundant_instance` or just `transport_index`?